src/olympia/hero/tests/test_models.py

```diff
@@ -42,8 +45,8 @@ def test_clean_requires_recommended(self):
 
     def test_clean_external_requires_homepage(self):
         ph = PrimaryHero.objects.create(
-            promoted_addon=PromotedAddon.objects.create(addon=addon_factory()),
-            is_external=True, gradient_color='#C60184', image='foo.png')
+            disco_addon=DiscoveryItem.objects.create(addon=addon_factory()),
+            is_external=True, gradient_color='#C60184', select_image=self.phi)
```

Comment:
and here - the import changes at the top of the file are likely incorrect now too.